packages/e2e/cypress/cli/cli.cy.ts

```diff
@@ -28,6 +28,7 @@ describe('CLI', () => {
                     PGUSER: 'postgres',
                     PGPASSWORD: Cypress.env('PGPASSWORD') || 'password',
                     PGDATABASE: 'postgres',
+                    PGSSLMODE: 'disable',
```

Comment:
these were using the default ssl mode rather than 'disable' - could you remove the changes in this file?